src/charm.py

```diff
@@ -870,6 +874,11 @@ def _apt_install(self, packages: Sequence[str]) -> None:
             execute_command(["dpkg", "--configure", "-a"])
             execute_command(["/usr/bin/apt-get", "install", "-qy"] + list(packages))
 
+    def _on_debug_ssh_relation_changed(self, _: ops.RelationChangedEvent) -> None:
+        """Handle debug ssh relation changed event."""
+        runner_manager = self._get_runner_manager()
+        runner_manager.flush()
```

Comment:
I'm guessing this is the method that doesn't remove runners that are already executing a job?

Reply:
Perhaps we can change the name of the function to indicate that it doesn't remove busy runners, like `flush_idle` or something

Reply:
it takes in an argument `flush_busy`, i think it should've been set to False to allow the busy runners to finish the jobs. Updating!